shared/profile/add-to-team/index.js

```diff
@@ -143,6 +143,7 @@ const AddToTeam = (props: Props) => {
               const selectedRole: TeamRoleType = (node && node.key) || null
               props.onRoleChange(selectedRole)
             }}
+            style={{width: 'auto'}}
```

Comment:
i'm not sure this works on mobile, can you check

Reply:
It does not. Fixing rn